src/shared/components/getting-started-section/index.js

```diff
@@ -1,30 +1,146 @@
+          <div className={ styles.hex2 }><HexSvg /></div>
+        </div>
+        <div className={ styles.content }>
+          <h1>{ messages.gettingStarted.sectionTitle }</h1>
+          <span className={ styles.sectionDescription }>
+            <p>{ messages.gettingStarted.sectionDesc }</p>
+          </span>
+          <div className={ styles.panel } >
+            <p className={ styles.liveSnippetTitle }>{messages.gettingStarted.addDataToIPFS}</p>
+            <LiveProvider key="add" className={styles.liveSnippet} code={codeAdd} scope={this.scopeAdd} mountStylesheet={false} transformCode={transformCode}>
+              <div className={ styles.liveSnippetEditorContainer }>
+                <LiveEditor ref={this.refAdd} className={'language-js ' + styles.liveSnippetEditor}/>
+                <button className={ styles.liveSnippetRun } onClick={this.handleRunClick('add')}>Run</button>
+              </div>
+              <div className={ styles.liveSnippetPreview } >
+                <p className={ styles.liveSnippetOutput }>{messages.gettingStarted.output}</p>
+                <pre>
+                  <code>{outputAdd}</code>
+                </pre>
+                {ipfsLoaded && <LivePreview/>}
+                {ipfsLoaded && <LiveError/>}
+              </div>
+            </LiveProvider>
+            <p className={ styles.liveSnippetTitle }>{messages.gettingStarted.getDataFromIPFS}</p>
+            <p className={ styles.liveSnippetSubtitle }>{messages.gettingStarted.usingJavascript}</p>
+            <LiveProvider key="get" className={ styles.liveSnippet } code={codeGet(cid)} scope={this.scopeGet} mountStylesheet={false} transformCode={transformCode}>
+              <div className={ styles.liveSnippetEditorContainer }>
+                <LiveEditor ref={this.refGet} className={'language-js ' + styles.liveSnippetEditor}/>
+                <button className={ styles.liveSnippetRun } onClick={this.handleRunClick('get')}>Run</button>
+              </div>
+              <div className={ styles.liveSnippetPreview } >
+                <p className={ styles.liveSnippetOutput }>{messages.gettingStarted.output}</p>
+                <pre>
+                  <code>{outputGet}</code>
+                </pre>
+                {(ipfsLoaded && cid) && <LivePreview/>}
+                {ipfsLoaded && <LiveError/>}
+              </div>
+            </LiveProvider>
+            <p className={ styles.liveSnippetSubtitle }>{messages.gettingStarted.usingCli}</p>
+            <div className={ styles.liveSnippetCliContainer }>
+              <SyntaxHighlighter codeStr={ `npm install ipfs -g
+jsipfs get ${cid}` } language='bash' />
+            </div>
+
+            <p className={ styles.liveSnippetSubtitle }>{messages.gettingStarted.usingGateway}</p>
+            <a className={ styles.liveSnippetLink } target="_blank" rel="noopener noreferrer" href={`https://ipfs.io/ipfs/${cid}`}>{`https://ipfs.io/ipfs/${cid}`}</a>
```

Comment:
You can use [Link](https://github.com/ipfs/js.ipfs.io/blob/master/src/shared/components/link/index.js) component here passing an `href` prop to it.